gradebook/src/GradeBook/book.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+
+/*
+The class Book packages the blueprint book provides ways to:
+1 store the grades. 
+2 add to the given gradebook obj
+3 welcomes the user from args
+4 Assign name to the gradebook
+ */
+
+namespace GradeBook
+{
+    public class book
+    {
+        List<double> grades;
```

Comment:
These should have an access modifier (`private`, `public`, etc).  Usually we prefer [properties](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties) for non-private fields (and sometimes even private).

Reply:
Oh, I actually presumed that the members without explicit access modifier would be private. But it makes sense to have it mentioned explicitly.

Reply:
.NET default access modifiers are a little tricky, it's better to do them explicitly everywhere.